azure-quantum/tests/integration/test_integration.py

```diff
@@ -25,36 +26,41 @@
     PathRelinkingSolver
 )
 
-
-def get_config() -> configparser.ConfigParser:
-    """Read config file and return config parser
-
-    :return: Config parser for reading config file
-    :rtype: configparser.ConfigParser
-    """
-    config = configparser.ConfigParser()
-    path = os.path.abspath(os.path.join(os.path.split(__file__)[0], "..", ".."))
-    config_path = os.path.join(path, "config.ini")
-    assert os.path.exists(config_path), "Cannot run integration tests: no config file found in azure-quantum folder."
-    config.read(config_path)
-
-    return config
-
-
 def create_workspace() -> Workspace:
```

Comment:
```suggestion
def create_workspace(resource_url: str = "https://quantum.microsoft.com") -> Workspace:
```

Reply:
The integration tests were merged into the unit tests and all applicable PEP8 fixes were applied with #53, #54, #56.